website/markdown/docs/usage/commands.mdx

```diff
@@ -22,7 +22,47 @@ The project is deleted automatically once you are done with editing. If you wish
 tuist edit --permanent
 ```
 
-That will generate a `Manfiest.xcodeproj` project that you can open manually.
+That will generate a `Manifest.xcodeproj` project that you can open manually.
+
+# Scaffold
+
+After project description helpers, `scaffold` is a next step to make it easier to create new modules in your app.
```

Comment:
I'd rephrase this a bit to not use "after xx" because that's a temporary reference that readers might not have context about. What about something along the lines of:

```
In projects with an established architecture, developers might want to bootstrap new components or features that are consistent with the project....
```